libraries/USB/src/USBHID.h

```diff
@@ -0,0 +1,79 @@
+// Copyright 2015-2020 Espressif Systems (Shanghai) PTE LTD
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+#include <stdint.h>
+#include <stdbool.h>
+#include "esp32-hal.h"
+#include "USB.h"
+#if CONFIG_TINYUSB_HID_ENABLED
```

Comment:
Any reason why both `CFG_TUD_HID` is checked in the cpp files but `CONFIG_TINYUSB_HID_ENABLED` is checked in headers? Shouldn't they be the same value?

Reply:
they generally should. done in https://github.com/espressif/arduino-esp32/pull/5538/commits/196ce947789c1d8f3a86047110049ab0edbef5ec